### PR TITLE
[Xnnpack] update doc

### PR DESCRIPTION
**Pull request by wejoncy**
### Description
- iOS support
- MatMul && GEMM
- Resize
- ConvTranspose

A deployment [link](https://wejoncy.github.io/onnxruntime/) for showcase.

### CODE REVIEW

File: docs/build/eps.md [hunk 1]
```diff
@@ -723,13 +723,13 @@ Once you have all the necessary components setup, follow the instructions to [cr
...
 
-Usage of XNNPACK on Android/Windows/Linux platforms is via the XNNPACK EP.
+Usage of XNNPACK on Android/Windows/Linux/iOS platforms is via the XNNPACK EP.
 
 See the [XNNPACK Execution Provider](../execution-providers/Xnnpack-ExecutionProvider.md) documentation for more details.
 
 The pre-built ONNX Runtime package([`onnxruntime-android`](https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-android)) for Android includes the XNNPACK EP.
 
-We don't support XNNPACK on iOS currently.
+We don't have a pre-built package for iOS currently, you can build it following the below steps if you want.
```
Comment (wejoncy) [inline docs/build/eps.md:732]: Need to ensure if we have pre-built packages

File: docs/execution-providers/Xnnpack-ExecutionProvider.md [hunk 2]
```diff
@@ -23,12 +23,14 @@ Accelerate ONNX models on Android devices and WebAssembly with ONNX Runtime and
 Pre-built packages of ONNX Runtime([`onnxruntime-android`](https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-android)) with XNNPACK EP for Android are published on Maven.
 See [here](../install/index.md#install-on-android) for installation instructions.
 
-There is no pre-built package for iOS yet. We will update it once it is ready.
+Pre-built binaries(`onnxruntime-objc` and `onnxruntime-c`) of ONNX Runtime with XNNPACK EP for iOS are published to CocoaPods.
+See [here](../install/index.md#install-on-ios) for installation instructions.
+
```
Comment (skottmckay) [inline docs/execution-providers/Xnnpack-ExecutionProvider.md:28]: This won't be available until the 1.14 release will it?

Comment (natke) [reply docs/execution-providers/Xnnpack-ExecutionProvider.md:28]: @wejoncy Can you clarify/correct this?

Comment (wejoncy) [reply docs/execution-providers/Xnnpack-ExecutionProvider.md:28]: Yeah, This won't be avaible until a new to support xnnpack ep is avaible. I am working on that PR.

Comment (wejoncy) [reply docs/execution-providers/Xnnpack-ExecutionProvider.md:28]: Sure. Added.

File: js/script.js [hunk 3]
```diff
@@ -1169,19 +1169,19 @@ var validCombos = {
         "Follow build instructions from <a href='https://aka.ms/build-ort-snpe' target='_blank'>here</a>",
 
     //Xnnpack
-    "android,C-API,ARM64,XNNPACK":
+    "ios,android,C-API,ARM64,XNNPACK":
```
Comment (faxu) [inline js/script.js:1172]: This is an invalid format. iOS entry will need a new entry. Please stage this for preview to test: https://github.com/microsoft/onnxruntime/wiki/Update-ONNX-Runtime-Docs#preview-change

**Review by faxu — CHANGES_REQUESTED** [review]

File: js/script.js (hunk 3 above)
Comment (wejoncy) [reply js/script.js]: Thanks.
Provided the [link](https://wejoncy.github.io/onnxruntime/) in description part.

**Review by natke — APPROVED** [review]

**Review by natke — COMMENTED** [review]
I think this still needs changes to the script.js file. Can you please check this?

**Review by faxu — APPROVED** [review]

**Pull request merged**